### DISCUSSION TITLE
Client gets disconnected after 30-60 seconds

### DISCUSSION THREAD

**Discussion by Nagol12344** [Totodore/socketioxide#360] [Q&A]
Hi!

Im having this weird issue where a client gets disconnected, and I get the error message "An existing connection was forcibly closed by the remote host." on the client, I am using rust_socketio as my client side library, and I need to have a contentiousness connection. 

EDIT: Sorry for making this, it was a issue with my client!

**Comment by Totodore** [comment]
> Hi!
> 
> Im having this weird issue where a client gets disconnected, and I get the error message "An existing connection was forcibly closed by the remote host." on the client, I am using rust_socketio as my client side library, and I need to have a contentiousness connection.

Coud you post a minimal reproductible examples as well as the configuration you used please ? (Http libs, socket Io version, library version, ...)
